### DISCUSSION TITLE
Warping streamlines with syn_registration mapping produces single point streamlines

### DISCUSSION THREAD

**Discussion by staunton-golding** [dipy/dipy#3062] [Tractography / Tractometry]
As the title says, when I attempt to warp my streamlines with the mapping return from the syn_registration method, I get single point streamlines. 

I am following the tutorial "streamline_registration," and everything looks good and works well, up until this point. The affine transformation of the streamlines will result in a regular nibabel.streamlines array sequence, with each array being multiple points. When visualized, these affine transformed streamlines look good, and look really close to the target image. However, the syn transformation of the streamlines results in a parsing of these array sequences, where the once grouped coordinates are now solo. 

I don't know if this is a bug. Any suggestions on how to approach this?

Best,
Hank

**Comment by skoudoro** [comment]
Hi @staunton-golding,

> However, the syn transformation of the streamlines results in a parsing of these array sequences, where the once grouped coordinates are now solo

Not sure to understand this sentence.  can you print `streamlines._offsets` and `streamlines._lengths`. 

are you using horizon to display the streamlines ? if not, which tool are you using and how do you save your streamlines ?
